### PR TITLE
Proper Logging on Errors.

### PR DESCRIPTION
**Pull request by varunsharma27**
**Before:**
image

**Now:**
image

PR Suggestion, use `this._logger`:
image

### CODE REVIEW

File: src/migrations/migration_runner.js [hunk 1]
```diff
@@ -76,10 +76,16 @@ export default class MigrationRunner {
...
-        warning += `${count} objects - "${migration.constructor.description}"\n`;
+      try {
+        const count = await migration.count();
+        toUpgrade += count;
+        if (count > 0) {
+          warning += `${count} objects - "${migration.constructor.description}"\n`;
+        }
+      } catch (e) {
+        this._logger.error(`Error during migration.count: ${migration.constructor.description}`);
+        console.error(e);
```
Comment (szydan) [inline src/migrations/migration_runner.js:87]: do we need this console?

Comment (varunsharma27) [reply src/migrations/migration_runner.js:87]: This is the LOC responsible for the stack trace. I'm not sure whether `this._logger.error` would also parse the error and print stack trace. I can try.

Comment (szydan) [reply src/migrations/migration_runner.js:87]: if it is needed than 
just add a comment that it is needed

Comment (varunsharma27) [reply src/migrations/migration_runner.js]: Looks beautiful enough with the `_logger`:
image
Pushed.

Comment (varunsharma27) [reply src/migrations/migration_runner.js]: As you can see, the error message is also being logged in the third last line, that is being [caught here](https://github.com/sirensolutions/kibi-internal/blob/14632fd19948df318acd6a27c7bd5859f6145163/src/cli/kibi/upgrade.js#L162).
Alternatively, we may also ```process.stderr.write(`${error}\n`);``` -> `logger.error(error);` over there and remove the logging of error object from here.

**Pull request merged**